lib/ruby-stackoverflow/client.rb

```diff
@@ -53,16 +42,20 @@ def parse_response(data, klass)
     def append_params_to_url(url, options)
       url = Configuration.api_url + url
       options.merge!(key_params)
-      options = options.to_a.map{|k,v|"#{k}=#{v}"}
-      url+'?'+options.join('&')
+      options = options.to_a.map { |k, v| "#{k}=#{v}" }
+      url + '?' + options.join('&')
     end
 
     def key_params
-      {key: configuration.client_key, site: 'stackoverflow', access_token: configuration.access_token}
+      {
+        key: configuration.client_key,
+        site: 'stackoverflow',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.